src/auth/src/credentials.rs

```diff
@@ -315,12 +315,46 @@ fn adc_well_known_path() -> Option<String> {
         .map(|root| root + "/.config/gcloud/application_default_credentials.json")
 }
 
+#[cfg(test)]
```

Comment:
I do not think this is accessible from other crates. This is what I did:

https://github.com/googleapis/google-cloud-rust/blob/66a07fe9b269a42eb00b5aebcefb221f4abfed11/src/auth/src/credentials.rs#L318-L337

Should we go with my PR?

Reply:
Your PR seems like a better idea.